src/kernels/normalkernel.jl

```diff
@@ -12,66 +12,43 @@ eltype(::AbstractNormalKernel{T}) where {T} = T
 
 Standard parametrisation of Normal kernels.
 """
-struct NormalKernel{T,U,V<:AbstractMatrix} <: AbstractNormalKernel{T}
+struct NormalKernel{T,U,V} <: AbstractNormalKernel{T}
     μ::U
     Σ::V
     function NormalKernel(μ, Σ)
         new{eltype(μ),typeof(μ),typeof(Σ)}(μ, Σ)
     end
 end
 
-"""
-    NormalKernel(Φ::AbstractMatrix, Σ::AbstractMatrix)
+const AffineNormalKernel{T} =
+    NormalKernel{T,<:AbstractAffineMap,<:Union{UniformScaling,AbstractMatrix}}
 
-Creates a Normal kernel with linear conditional mean of slope Φ and covariance matrix Σ.
-"""
-NormalKernel(Φ::AbstractMatrix, Σ::AbstractMatrix) = NormalKernel(AffineMap(Φ), Σ)
+const AffineIsoNormalKernel{T} = NormalKernel{T,<:AbstractAffineMap,<:UniformScaling}
```

Comment:
This is not used _anywhere_. Is it really needed?

Reply:
Huh. Maybe not?